src/packages/README.md

```diff
@@ -1,21 +1,85 @@
 # Third-party packages
 
-CheriBSD on Morello ships with both hybrid and CheriABI packages
+CheriBSD on Morello ships with both hybrid ABI and CheriABI packages
 (compilations) of the FreeBSD ports collection, each targeting a different
 form of code generation and Application Binary Interface (ABI).
 They have different levels of completeness, maturity, security, and support.
 
+## Overview
+
+The following table presents an overview of available package managers in
+CheriBSD that are described in more details in consecutive sections.
+
+| ABI        | #      | Manager | Install path   | Suitable for     | Examples |
+|------------|--------|---------|----------------|------------------|----------|
+| Hybrid ABI | 20,000 | `pkg64` | `/usr/local64` | Day-to-day use   | bash<br/>gdb-cheri<br/>llvm-base<br/>vim |
+| CheriABI   | 8,000  | `pkg64c`| `/usr/local`   | Experimental use | dash<br/>git<br/>nano<br/>rsync |
```

Comment:
I wonder if we should somehow more explicitly flag the # column as being approx. I guess ‘Approx #’?

Reply:
Width of the text in the guide is quite limited and using "Approx #" would result in a line break. I preceded numbers with "~". What do you think about this?

We could also consider increasing the width.

Reply:
Seems good.